spi/protocol-spi/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+val jacksonVersion: String by project
```

Comment:
unused variable

Reply:
@bscholtes1A can you make this change and I'll merge?

Reply:
done